docs/pages/admin-guides/management/guides/github-integration.mdx

```diff
@@ -0,0 +1,148 @@
+repositories using short-lived SSH certificates signed by an approved CA, such
+as a Teleport CA. Optionally, organizations can enforce stricter security by
+requiring these signed SSH certificates for access, effectively disabling the
+use of personal SSH keys and access tokens.
+
+Teleport users can configure their Git repositories to proxy through Teleport.
+After setup, Git commands automatically route through Teleport, which
+impersonates their GitHub identities using short-lived SSH certificates signed
+by Teleport's CA for authentication with GitHub. Each Git command proxied
+through Teleport is also logged in Teleport's audit events.
+
+To retrieve a user's GitHub identity, `tsh` initiates the GitHub OAuth flow by
+opening a browser window for the user to log in with their GitHub credentials.
+
+![GitHub SSH certificate authorities](../../../../img/management/how-it-works-github-proxy.svg)
+
+Note that Teleport proxies Git commands through SSH but the users should
+continue to access github.com website regularly through their browsers.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs-enterprise.mdx version="17.2"!)
+- Access to GitHub Enterprise and permissions to modify GitHub's SSH certificate
+  authorities and configure OAuth applications.
+- (!docs/pages/includes/tctl.mdx!)
+- Git locally installed
+
+## Step 1/4. Configure a GitHub OAuth application
+
+The GitHub integration requires a GitHub OAuth application to obtain users'
+GitHub identities. You can skip this step if the Teleport users use GitHub SSO
+to sign in Teleport.
+
+Go to "OAuth Apps" under "Developer Settings" of your organization's settings.
+Click on "New OAuth App".
+
+Fill in the details. Use the following for "Authentication callback URL":
+```
+https://<Var name="teleport-proxy-address"/>/v1/webapi/github/
+```
+
+Once the OAuth application is created, create a client secret and remember the
+client ID and the secret for the next step:
+
+![A GitHub Oauth App, highlighting Client ID and secret](../../../../img/sso/github-app-vars.png)
+
+## Step 2/4. Create a GitHub integration and export the CAs
+
+Now create a yaml file that represents the Github integration:
+```yaml
+# github_integration.yaml
+kind: integration
+sub_kind: github
+version: v1
+metadata:
+  name: github-<Var name="my-github-org"/>
+spec:
+  github:
+    organization: <Var name="my-github-org"/>
+  credentials:
+    id_secret:
+      id: <Var name="oauth-app-client-id"/>
+      secret: <Var name="oauth-app-client-secret"/>
+```
+Replace `my-github-org` with the organization name, and replace
+`oauth-app-client-id` and `oauth-app-client-secret` with values from the
+previous step.
+
+To create the resource with `tctl`, run:
+```bash
+tctl create -f github_integration.yaml
+```
+
+Once the integration resource is created, export the CA to be used for GitHub:
+```bash
+tctl auth export --type github --integration github-<Var name="my-github-org"/>
+```
```

Comment:
````suggestion
```code
$ tctl auth export --type github --integration github-<Var name="my-github-org"/>
```
````

So we can use our custom code snippet component here